kclvm/driver/src/kpm_update.rs

```diff
@@ -0,0 +1,96 @@
+use anyhow::{bail, Result};
+use std::{collections::HashMap, env, iter, path::PathBuf, process::Command};
+
+const MANIFEST_FILE: &str = "kcl.mod";
+
+/// Update the KCL module.
+///
+/// This function calls `kcl mod update` to update the KCL module.
+pub(crate) fn update_kcl_module(manifest_path: PathBuf) -> Result<()> {
+    match lookup_the_nearest_file_dir(manifest_path.clone(), MANIFEST_FILE) {
+        Some(mod_dir) => {
+            match Command::new(kcl())
+                .arg("mod")
+                .arg("update")
+                .current_dir(mod_dir)
+                .output()
+            {
+                Ok(output) => {
+                    if !output.status.success() {
+                        bail!(
+                            "update failed with error: {}",
+                            String::from_utf8_lossy(&output.stderr)
+                        );
+                    }
+                    Ok(())
+                }
+                Err(err) => bail!("update failed with error: {}", err),
+            }
+        }
+        None => bail!("Manifest file '{}' not found in directory hierarchy", MANIFEST_FILE),
+    }
+}
+
+/// Get the path for the KCL executable.
+fn kcl() -> PathBuf {
```

Comment:
This method is the same as https://github.com/kcl-lang/kcl/blob/7232ff1346a91d23bf97377f5a34a67aa188109f/kclvm/driver/src/kpm_metadata.rs#L140

You can move the method `kcl()` to file https://github.com/kcl-lang/kcl/blob/main/kclvm/driver/src/lib.rs.
And reuse it in both `kpm_metadata.rs` and `kpm_update.rs`

Reply:
okay